engine/src/main/java/org/pentaho/di/trans/steps/jobexecutor/JobExecutor.java

```diff
@@ -222,6 +222,7 @@ private void executeJob() throws KettleException {
 
     ExtensionPointHandler.callExtensionPoint( log, KettleExtensionPoint.JobStart.id, data.executorJob );
 
+    data.executorJob.setInitialLogBufferStartLine();
```

Comment:
Why you need this here? The method execute on job is not call?

Reply:
Some jobs aren't started from the execute method